collection-scripts/gather_network_logs

```diff
@@ -253,10 +255,10 @@ elif [[ "${NETWORK_TYPE}" == "kuryr" ]]; then
     gather_kuryr_nodes_data
     gather_kuryr_data
 elif [[ "${NETWORK_TYPE}" == "ovnkubernetes" ]]; then
-    if [-z "$IPSEC_PODS"]; then
+    if [ -z "$IPSEC_PODS" ]; then
         echo "INFO: No ovn-ipsec pods exist, tunnel traffic will be unencrypted"
     else 
-        echo "INFO: ovn-ipsec is enabled, tunnel traffic should be encryted"  
+        echo "INFO: ovn-ipsec is enabled, tunnel traffic should be encrypted"
```

Comment:
With this being true is running `gather_ovn_ipsec_data` valuable? 
Will we be decrypting it to use what we collect? If we do decrypt it do we need to tell the user? It doesn't look like we collect any traffic only how we shape/direct the traffic so these question may not matter, and if so I don't know if the messages/logs are needed (but they are not harmful).